### DISCUSSION TITLE
Welcome to perf-ninja Discussions!

### DISCUSSION THREAD

**Discussion by dendibakh** [dendibakh/perf-ninja#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about lab assignments.
  * Share ideas and suggestions.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by lgreydev** [comment]
@dendibakh. Thanks for your work!

**Reply by dendibakh** [reply]
You're welcome
